File: day014/day014.c
#include <stdio.h>

int main()
{
    int x, y;
    
    printf("Bir sayı giriniz: ");
    scanf("%d", &x);

    printf("Bir sayı giriniz: ");
    scanf("%d", &y);
    
    if (x % y == 0 || y % x == 0)
        printf("Girdiğiniz sayılar birbirlerinin katlarıdır.");
    else
        printf("Girdiğiniz sayılar birbirlerinin katları değildir.");
    
    return 0;
}

File: day014/day014.java
public class Day014
{
	public static void main(String[] args) {
		int x, y;
		java.util.Scanner kb = new java.util.Scanner(System.in);
		
		System.out.print("Bir sayı giriniz: ");
		x = kb.nextInt();

		System.out.print("Bir sayı daha giriniz: ");
		y = kb.nextInt();
		
        	if (x % y == 0 || y % x == 0) 
            		System.out.println("Girdiğiniz sayılar birbirlerinin katlarıdır.");
        	else
            		System.out.println("Girdiğiniz sayılar birbirlerinin katları değildir.");
	}
}
